backend/Ript-Fitness-Backend/src/main/java/com/riptFitness/Ript_Fitness_Backend/domain/repository/ExerciseRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.riptFitness.Ript_Fitness_Backend.domain.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import com.riptFitness.Ript_Fitness_Backend.domain.model.ExerciseModel;
+
+public interface ExerciseRepository extends JpaRepository <ExerciseModel, Long> {
+	
+	// Query to retrieve exercises by account_id where isDeleted is false
+    @Query("SELECT e FROM ExerciseModel e WHERE e.account.id = :accountId AND e.isDeleted = false")
+    List<ExerciseModel> findByAccountIdAndNotDeleted(Long accountId);
```

Comment:
Probably more of a Tina thing, but can you change the method name to just "findById" and override the JPA method? I know other features call it findById even if it's checking for isDeleted to be false. You can look at my FoodRepository to see how I did it (again, more of a Tina thing), but this will keep things uniform among the features :)